### PR TITLE
Update current actions

### PR DESCRIPTION
**Pull request by doomspork**
1. Update the content deployment workflow to use the `master` branch
2. Remove the current `ci` workflow
3. Remove extra files

### CODE REVIEW

**Review by kinson — APPROVED** [review]
🎉

**Comment by doomspork** [comment]
Sorry @kinson I pushed a comment just after you reviewed, dismissing it.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,38 +2,39 @@
 # Order is important. The last matching pattern takes the most precedence.
 
 # Default owners for everything in the repo.
-*        @elixirschool/developers
+*        @elixirschool/english
 
-en/      @elixirschool/english
+/lessons/en/      @elixirschool/english
```
Comment (kinson) [inline .github/CODEOWNERS:7]: since it was `en/` before, should the new path be `lessons/en/` or do both work?

**Review by kinson — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (doomspork) [reply .github/CODEOWNERS:7]: I _think_ we just need the latter since the former no longer exists. Thoughts?

Comment (kinson) [reply .github/CODEOWNERS:7]: Sorry for the confusion, I was just referring to the leading slash 😅 

So:

`/lessons/en` vs
`lessons/en`

Comment (doomspork) [reply .github/CODEOWNERS:7]: Oh! Yeah, that makes sense

Comment (kinson) [reply .github/CODEOWNERS:7]: I'm not sure if it makes a difference tbh so if you want to merge this and then check that works!

**Comment by doomspork** [comment]
We should figure out a CI pipeline for this repo later but that doesn't need to block us finishing other things.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,38 +2,38 @@
...
+lessons/en/      @elixirschool/english
 
-es/      @elixirschool/spanish
+lessons/es/      @elixirschool/spanish
 
-fr/      @elixirschool/french
+lessons/fr/      @elixirschool/french
 
-gr/      @elixirschool/greek
+lessons/gr/      @elixirschool/greek
```
Comment (kinson) [inline .github/CODEOWNERS:13]:
```suggestion
lessons/el/      @elixirschool/greek
```

Since we just updated gr -> el

**Review by kinson — CHANGES_REQUESTED** [review]
Whoops missed this on the first pass

**Comment by doomspork** [comment]
@kinson if it’s GTG please merge when you approve 🙏

**Review by kinson — APPROVED** [review]

**Pull request merged**